examples/advanced/federated-statistics/hierarchical_stats/demo/visualization.ipynb

```diff
@@ -0,0 +1,222 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "7c4be7b0",
+   "metadata": {},
+   "source": [
+    "# NVFLARE Hierarchical Statistics Visualization"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "987e6028",
+   "metadata": {},
+   "source": [
+    "#### dependencies\n",
+    "\n",
+    "To run examples, you might need to install the dependencies\n",
+    "* ipywidgets\n",
+    "* numpy\n",
+    "* pandas\n",
+    "* matplotlib\n",
+    "* jupyter\n",
+    "\n",
+    "These are captured in the requirements.txt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "665dc17e",
+   "metadata": {},
+   "source": [
+    "## Hierarchical Statistics Visualization of Tabular Data\n",
+    "In this example, we demonstrate how to visualize the hierarchical results from the statistics of tabular data. The visualization requires json, pandas, matplotlib and ipywidgets modules. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 71,
+   "id": "c44a0217",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import ipywidgets as widgets\n",
+    "import json\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "30c79d1a",
+   "metadata": {},
+   "source": [
+    "First, copy the resulting json file to the demo directory. In this example, resulting file is called hierarchical_stats.json. Then load the json file"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 72,
+   "id": "44f6bed2",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "with open('hierarchical_stats.json', 'r') as f:\n",
+    "    data = json.load(f)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1b0f21fd",
+   "metadata": {},
+   "source": [
+    "### Overall Statistics\n",
+    "\n",
+    "In this section, we demonstrate how to visualize hierarchical stats from the tabular data."
```

Comment:
suggest: "In this section, we demonstrate how to visualize hierarchical stats from the tabular data using Jupyter Widgets."

Reply:
Done, thanks.